server/homebrew.api.js

```diff
@@ -57,22 +57,23 @@ const api = {
 						googleError = err;
 					});
 				// Throw any error caught while attempting to retrieve Google brew.
-				if(googleError) throw googleError;
+				if(googleError) {
+					const reason = googleError.errors[0].reason;
+					throw { ...googleError, HBErrorCode: reason == 'notFound' ? '02' : '01' };
```

Comment:
Similar to below comment, can we make split 02 into two to help users troubleshoot these different scenarios?

- The user is logged in, and this brew is under their account (you deleted your google drive files by accident!)
- The user is not logged in, or this brew is not under their account (this brew is owned by a different account, which explains why you don't see a record of it in your google drive. Are you logged into the right account?)

Reply:
At this stage, I don't think we can - I haven't been able to spot any way to differentiate "you don't have access" from "this doesn't exist". I'll leave this unresolved for now, in case we have any flashes of genius later on.

Reply:
The error is the same (brew doesn't exist), so the response from Google would be the same.

What I'm asking is to add a little more info to differentiate:

- This brew doesn't exist, *and you own it* as first author
- This brew doesn't exist, *and you don't own it*


to help explain the cases where users made the brew on an old Google Drive account and then switched to a different login and have continued editing as an invited author. I.e., to track down which account deleted the brew.

Reply:
This should be in the latest commit.